pkg/plugins/generic/generic_plugin_test.go

```diff
@@ -0,0 +1,94 @@
+package generic
+
+import (
+	"testing"
+
+	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
Any chance to use ginkgo and gomega as we do in other places  ?

also you could define a test package (i.e `generic_test`) and just test the interface calls WDYT ?

Reply:
done